postgres-driver/application.go

```diff
@@ -85,6 +85,7 @@ type dbApplication struct {
 	PAAdress             sql.NullString `db:"pa_address"`
 	SecretKey            sql.NullString `db:"secret_key"`
 	URL                  sql.NullString `db:"url"`
+	EnterpriseLimit      sql.NullInt64  `db:"enterprise_limit"`
```

Comment:
Could you help clarify the reason for surfacing this field at application level? There is already a PayPlanType to handle different types of limits.